test/mutable.js

```diff
@@ -1,7 +1,7 @@
 const crypto = require('crypto');
 const should = require('should');
 const h = require('./helpers');
-const consts = require('../src/consts');
+const consts = require('../src/consts').pubConsts;
```

Comment:
Can be refactored as `const {pubConsts: consts} = require('../src/consts')`

Reply:
We can bring the import refactoring as a separate task

Reply:
I'll include those changes in this PR